python/feathub/processors/local/local_processor.py

```diff
@@ -178,26 +211,41 @@ def materialize_features(
                 timestamp_field=features.timestamp_field,
                 timestamp_format=features.timestamp_format,
             )
-        elif isinstance(sink, FileSystemSink):
+        elif isinstance(sink, FileSystemSink) and _is_local_file_or_dir(sink.path):
             insert_into_file_sink(features_df, sink)
             return LocalJob()
 
-        raise RuntimeError(f"Unsupported sink: {sink}.")
+        self._init_spark_session_local_mode()
+        try:
+            return self._insert_into_spark_supported_sink(
```

Comment:
It seems unnecessary to specify `supported` here. After all, we do not explicitly check if the sink is supported by spark.

How about `_materialize_dataframe_using_spark`?